spec.md

```diff
@@ -380,13 +380,43 @@ Example error response (invalid `id` provided):
 
 ## Collections
 
+A collection is a namespace for Resources, and a unit of configuration, within
+a space.
+
+In other storage systems, a collection is also known as: Directory, Folder,
+RDBMS Table, Document Collection, Graph, WebAPI FileList, Bucket, Solid
+Container, EDV Vault, DWN Collection, and so on.
+
+The use of collections is optional -- when a Space is created, the default `/`
```

Comment:
```suggestion
The use of collections is optional -- when a Space is created, the default
```

can we not commit to implying the root collection is named `/`?

Reply:
Oh, sure, no prob.